presto-redis/src/main/java/io/prestosql/plugin/redis/RedisPlugin.java

```diff
@@ -13,35 +13,21 @@
  */
 package io.prestosql.plugin.redis;
 
-import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableList;
 import io.prestosql.spi.Plugin;
 import io.prestosql.spi.connector.ConnectorFactory;
-import io.prestosql.spi.connector.SchemaTableName;
 
-import java.util.Map;
 import java.util.Optional;
-import java.util.function.Supplier;
-
-import static java.util.Objects.requireNonNull;
 
 /**
  * Presto plugin to use Redis as a data source.
  */
 public class RedisPlugin
         implements Plugin
 {
-    private Optional<Supplier<Map<SchemaTableName, RedisTableDescription>>> tableDescriptionSupplier = Optional.empty();
```

Comment:
Best would be to apply same approach as in Kafka. https://github.com/prestosql/presto/pull/6046 where `tableDescriptionSupplier` is some form of metastore (like HMS in Hive connector). WDYT?

Reply:
I think this could be a nice follow up

Reply:
Please capture this as issue so some one could follow up.

Reply:
can you please do it? you'll better describe this. and i actually do not want this change to happen :)